File: ComercioEletronico/Python/categoria.py
import json

class Categoria:
    def __init__(self, id, d):
        self.set_id(id)
        self.set_descricao(d)
        
    def set_id(self, id):
        self.__id = id
    
    def set_descricao(self, d):
        self.__d = d

    def get_id(self):
        return self.__id

    def get_descricao(self):
        return self.__d

    def __str__(self):
        return f"{self.__id} - {self.__d}"

File: ComercioEletronico/Java/src/Categoria.java
public class Categoria{
    private int id;
    private String descricao;

    public Categoria(int id, String d){
        setId(id);
        setDescricao(d);
    }

    public void setId(int id){
        this.id = id;
    }

    public void setDescricao(String d){
        this.descricao = d;
    }

    public int getId(){
        return this.id;
    }

    public String getDescricao(){
        return this.descricao;
    }

    public String toString(){
        return String.format("Categorias\n ID = %s - Descrição = %s", id, descricao);
    }
}